### DISCUSSION TITLE
[@types/intercom-web] IntercomCommandSignature doesn't contain reattach_activator

### DISCUSSION THREAD

**Discussion by glenngijsberts** [DefinitelyTyped/DefinitelyTyped#64258] [Issues with a @types package] [answered]
I'm working on migrating some code related to Intercom to TypeScript, and after installing the [@types/intercom-web](https://www.npmjs.com/package/@types/intercom-web) package, I noticed that one of the commands that are being used in the initialization script is not added to the commands [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/intercom-web/index.d.ts#L50). It's the `reattach_activator` that's missing and is giving me errors. Is this one excluded on purpose?

Intercom script can be found [here](https://developers.intercom.com/installing-intercom/docs/basic-javascript#how-to-install).

[a comment by an automated account is not shown]

**Comment by onatm** [comment, marked as answer]
> It's the `reattach_activator` that's missing and is giving me errors. Is this one excluded on purpose?

It's definitely not excluded on purpose. Most of the contributors added the existing parameter and function definitions when they needed them. I'd be happy if you could cut a PR for it @glenngijsberts

**Comment by glenngijsberts** [comment]
@onatm Thanks a lot! Just created https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64264. Let me know if I did anything wrong.

**Reply by onatm** [reply]
Thank you for the contribution @glenngijsberts 🎉
